Resources/Textures/_RMC14/Objects/Clothing/OuterClothing/Coats/CO/standardissuejackets/cojacketgrey.rsi/meta.json

```
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

```suggestion
  "license": "CC-BY-SA-4.0",
```
If you made this yourself it should be updated to 4.0